core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -248,9 +248,16 @@
       "The number of threads to use when coordinating a bulk import."),
   MASTER_BULK_TIMEOUT("master.bulk.timeout", "5m", PropertyType.TIMEDURATION,
       "The time to wait for a tablet server to process a bulk import request"),
+  MASTER_RENAME_THREADS("master.rename.threadpool.size", "20", PropertyType.COUNT,
+      "The number of threads to use when renaming user files during table import or bulk ingest."),
+  @Deprecated
+  @ReplacedBy(property = MASTER_RENAME_THREADS)
   MASTER_BULK_RENAME_THREADS("master.bulk.rename.threadpool.size", "20", PropertyType.COUNT,
       "The number of threads to use when moving user files to bulk ingest "
           + "directories under accumulo control"),
+  MASTER_IMPORTTABLE_RENAME_THREADS("master.importtable.rename.threadpool.size", "20",
```

Comment:
Can this be removed?

Reply:
So sorry, I could have sworn I took care of that. Will address. Sorry again!

Reply:
Done.